METRICS.md

```diff
@@ -51,14 +51,21 @@ The prefix of metric's namespace is `/intel/docker/<docker_id_or_root>/stats/cgr
 Namespace | Data Type | Description
 ----------|-----------|-----------------------
 cpu_stats/cpu_usage/total_usage | uint64 | The total CPU time consumed
-cpu_stats/cpu_usage/usage_in_kernelmode | uint64 | CPU time consumed by tasks in system (kernel) mode
-cpu_stats/cpu_usage/usage_in_usermode | uint64 | CPU time consumed by tasks in user mode
-cpu_stats/cpu_usage/percpu_usage/\<N\> | uint64 | CPU time consumed on each N-th CPU by all tasks
-cpu_stats/throttling_data/periods | uint64 | The number of period intervals that have elapsed
-cpu_stats/throttling_data/throttled_periods | uint64 | The number of times tasks in a cgroup have been throttled
+cpu_stats/cpu_usage/kernel_mode | uint64 | CPU time consumed by tasks in system (kernel) mode
+cpu_stats/cpu_usage/user_mode | uint64 | CPU time consumed by tasks in user mode
+cpu_stats/cpu_usage/per_cpu/\<N\>/value | uint64 | CPU time consumed on each N-th CPU by all tasks
+cpu_stats/throttling_data/nr_periods | uint64 | The number of period intervals that have elapsed
+cpu_stats/throttling_data/nr_throttled | uint64 | The number of times tasks in a cgroup have been throttled
 cpu_stats/throttling_data/throttled_time | uint64 | The total time duration for which tasks in a cgroup have been throttled
+cpu_stats/cpu_shares | uint64 | contains an integer value that specifies a relative share of CPU time available to the tasks in a cgroup
```

Comment:
To keep consistency, consider changing `contains an integer value that specifies a relative share of CPU time available to the tasks in a cgroup` to `The relative share of CPU time available to the tasks in a cgroup`. Please, take into account the rest of similar cases.